pyvista/core/pointset.py

```diff
@@ -1370,8 +1385,31 @@ def _from_arrays(
         >>> import numpy as np
         >>> from pyvista import CellType
         >>> import pyvista
-        >>> cells = np.array([8, 0, 1, 2, 3, 4, 5, 6, 7, 8, 8, 9, 10, 11, 12, 13, 14, 15])
-        >>> cell_type = np.array([CellType.HEXAHEDRON, CellType.HEXAHEDRON], np.int8)
+        >>> cells = np.array(
+        ...     [
+        ...         8,
```

Comment:
This is very un-black-like so I doubt this is supported (but I'm only guessing, to be clear). These are the kinds of uglies that are the price to pay for a "dumb" autoformatter. I wonder if it would look a bit less bad (or any different at all) without the "magic trailing comma"...

Reply:
Recommending using `np.hstack`.

Reply:
Fixed in 062fe1ab41418178050ae4121a75a2442a22f5f7